ansible/files/gotrue.service.j2

```diff
@@ -4,7 +4,7 @@ Description=Gotrue
 [Service]
 Type=simple
 WorkingDirectory=/opt/gotrue
-ExecStart=/opt/gotrue/gotrue
+ExecStart=/opt/gotrue/gotrue --config-dir /etc/auth.d
```

Comment:
Existing version already accepts this flag?

Reply:
Yes I've verified by logging into a project and making sure the flag and expected version are there.